src/components/WebexData/WebexData.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const AdapterContext = React.createContext();
+
+export default function WebexData(props) {
```

Comment:
Since this is a new component, use the spread operator for the props like we agreed on Adam's PR.
```suggestion
export default function WebexData({adapter, children}) {
```